index.js

```diff
@@ -80,6 +80,7 @@ function renderStoreIcons() {
     button.addEventListener("click", function() {
       addFruitToCartState(item)
       renderUserCart()
+      renderTotalPrice()
```

Comment:
Creating functions and using them where needed is very easy way to do the challenge. very nice approach.

Reply:
Thanks Abul! This is the first time that I feel I've got a good grasp on using functions and why they can be really powerful.